code/modules/reagents/reagents/medicine.dm

```diff
@@ -682,19 +682,19 @@
 	metabolism = REM/2
 
 /datum/reagent/medicine/detox/affect_blood(mob/living/carbon/M, alien, effect_multiplier)
-	if(M.metabolism_effects.nsa_threshold == initial(M.metabolism_effects.nsa_threshold))
-		M.metabolism_effects.nsa_threshold += rand(20, 60)
+	if(M.metabolism_effects.nsa_threshold_base == initial(M.metabolism_effects.nsa_threshold_base))
```

Comment:
this is problematic and gives problems with respect to perks.

Reply:
It is, but this chem is not my invention, it's been there before, and I just made sure that it works with new vars.

Reply:
I understand, this is just a comment and not a request for changes anyway.